optimize/backend/src/main/java/io/camunda/optimize/dto/optimize/query/LicenseInformationResponseDto.java

```diff
@@ -1,21 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.dto.optimize.query;
-
-import io.camunda.optimize.dto.optimize.OptimizeDto;
-import java.io.Serializable;
-import java.time.OffsetDateTime;
-import lombok.Data;
-
-@Data
-public class LicenseInformationResponseDto implements OptimizeDto, Serializable {
```

Comment:
did you update the REST API docs for this removal too?

Reply:
Tragically this was the first time I remembered, yet there is no docs for this on confluence